src/ring_theory/noetherian.lean

```diff
@@ -69,6 +69,18 @@ theorem fg_def {N : submodule R M} :
   exact ⟨t, rfl⟩
 end⟩
 
+lemma fg_iff_exists_fin_generating_fam {N : submodule R M} :
```

Comment:
I don't think this matter much, but mathlib seems to use `_family` but never `_fam`.
```suggestion
lemma fg_iff_exists_fin_generating_family {N : submodule R M} :
```